tests/utils/test_pynsee_utils.py

```diff
@@ -127,7 +124,11 @@ def test_dummy_sirene_token_is_not_stored(self):
         with open(config_file, "w") as f:
             json.dump({"sirene_key": "spam"}, f)
 
-        init_conn(sirene_key="eggs")
+        init_conn(
+            sirene_key="eggs",
+            http_proxy=os.environ.get("http_proxy"),
+            https_proxy=os.environ.get("https_proxy"),

```

Comment:
this should not be necessary, the env variables are checked in the session object, aren't they?

Reply:
That's to make the tests work locally when you're behind a proxy.